src/styles/_variables.scss

```diff
@@ -1,5 +1,4 @@
 // ====== COLOURS ========
-$primary-blue: #e3f2fd;
-$secondary-blue: #2d4b77;
-$white: #fff;
-$grey: #525252;
+$primary-beige: #f1e6cc;
+$primary-blue: #3f6476;
+$secondary-blue: #001f3f;
```

Comment:
non-blocking because I realizes I don't remember if the 3rd blue is used at all but there is one more color in the  design incase we want to add it in this push

#356481

Reply:
PRAISE: Three colors are good to start with (some palette websites generate palettes of four, five or more colors)! I like how we are getting rid of white!

Reply:
Yeah I didn't see the fourth colour used in any of the elements, so didn't add it in. But we can definitely add that later on if needed.